disco_aws_automation/disco_metanetwork.py

```diff
@@ -118,18 +115,40 @@ def _create_security_group(self):
         return security_group
 
     @property
-    def subnets(self):
-        '''Finds or creates the subnets for our metanetwork'''
-        if not self._subnets:
-            self._subnets = find_or_create(
-                self._find_subnets, self._create_subnets
-            )
-        return self._subnets
+    def disco_subnets(self):
+        '''Creates the subnets for our metanetwork'''
+        if not self._disco_subnets:
+            self._disco_subnets = self._create_subnets()
```

Comment:
This method is really just instantiating the DiscoSubnet objects, it doesn't actually create the subnets in AWS. There is logic inside that class that just loads up the subnet if it already there.